src/main/java/net/neoforged/neoforge/registries/datamaps/DataMapsUpdatedEvent.java

```diff
@@ -0,0 +1,89 @@
+/*
+ * Copyright (c) NeoForged and contributors
+ * SPDX-License-Identifier: LGPL-2.1-only
+ */
+
+package net.neoforged.neoforge.registries.datamaps;
+
+import java.util.function.Consumer;
+import net.minecraft.core.Registry;
+import net.minecraft.core.RegistryAccess;
+import net.minecraft.network.Connection;
+import net.minecraft.resources.ResourceKey;
+import net.neoforged.bus.api.Event;
+import org.jetbrains.annotations.ApiStatus;
+
+/**
+ * Event fired on the {@link net.neoforged.neoforge.common.NeoForge#EVENT_BUS game event bus} when the data maps of
```

Comment:
Either the event docs or the docs on the `SERVER_RELOAD` cause should mention that this fires from the apply phase of a reload listener and not after the reload is completed as this has implications on what other things you can touch from the event listener